h2o-automl/src/main/java/water/automl/api/schemas3/AutoMLBuildSpecV99.java

```diff
@@ -293,6 +293,10 @@ public static final class AutoMLBuildModelsV99 extends SchemaV3<AutoMLBuildSpec.
     @API(help = "A mapping representing monotonic constraints. Use +1 to enforce an increasing constraint and -1 to specify a decreasing constraint.",
             level = API.Level.secondary)
     public KeyValueV3[] monotone_constraints;
+    
+    @API(help = "Temporary flag to disable pipeline usage when preprocessing is enabled.", 
+            level = API.Level.expert)
+    public boolean pipelineEnabled;
```

Comment:
This seems to be impossible to use using the python API. Is it intentional?

Reply:
good catch, but as I'm removing the `AutoMLBuildSpec.pipelineEnabled` property in the cleanup PR (https://github.com/h2oai/h2o-3/pull/16061/) and I also forgot to remove it from the schema there, I'll do the change in that last PR if you don't mind.

Reply:
yes, that's fine.